source/features/filter-pr-by-build-status.tsx

```diff
@@ -34,7 +35,13 @@ function populateDropDown({currentTarget}: Event): void {
 	}
 }
 
-function init(): void | false {
+async function init(): Promise<void | false> {
+	const ciStatusIcon = await fetchCIStatus();
+
+	if (!ciStatusIcon) {
+		return false;
+	}
+
 	const reviewsFilter = select('.table-list-header-toggle > details:nth-last-child(3)')!;
```

Comment:
Since https://github.com/sindresorhus/refined-github/pull/2114 removed some filters, we might need to change the way we find the position.

It may work before this PR because this check is added before `clean-issue-filters` runs

Reply:
Given the structure, I don't think it will be possible to detect "Reviews" filter by just using CSS. We can detect the reviews filter using JavaScript and add the build status filter just after that? What do you say?

Reply:
Hey, "Assignee" and "Sort" filters are always there at the last and they are not getting removed. So this approach won't break.